src/img/icons/nft-icon.svg

```diff
@@ -0,0 +1,1945 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is this icon a placeholder? Otherwise, this needs to be formatted the way we format the rest of our icons

Reply:
So, I checked the Figma files and it seems like we have this screen that I don't remember seeing before so it might be new:

![FireShot Capture 670 - Safe Control – Figma - www figma com](https://user-images.githubusercontent.com/18473896/180068453-fb9136b6-cf28-4190-8fcf-84e648044aaa.png)

We could use that icon in the nft picker as a placeholder for when a NFT has not been selected, designs [here](https://www.figma.com/file/LlVmeMQxHVA04evfrgZyuN/Safe-Control?node-id=4924%3A22044)

What do you think?

Reply:
@ArmandoGraterol @arrenv I would expect to see this placeholder when an image/NFT could not be found, rather than not selected.